src/utils.js

```diff
@@ -62,12 +66,9 @@ export async function readPackage() {
 
 export async function getPlaywrightInstance(browserType) {
   const playwrightPackage = await readPackage()
-  if (playwrightPackage === 'playwright') {
```

Comment:
I'm not quit sure that it wiil be ok, if we are using `playwright-firefox` for example, and then we will run tests without browser definition, that's ok for now, but now it will throw an error, i suppose, cause `chromium` will be default value. So we need some extra logic to find out using version

Reply:
we = jest tests in this repo?

yep lets maybe figure out a better way later if we want to test the individual configurations.
Tests are fixed now with this workaround.

Reply:
@mxschmitt no. It will crash for other users with it configuration:
**package.json**
```json
...
"scripts": {
    "test": "jest"
  },
"dependencies": {
    "jest-playwright-preset": "0.0.9",
    "playwright-firefox": "^0.11.0"
  },
...
```

With **0.10.0** running `test` will be fine, but with **0.11.0** it will crash I suppose, if browser not be defined. So it need to be `BROWSER=firefox test` as example

Reply:
I can check it out and just add some extra logic in this PR

Reply:
would be awesome, thanks